Add docker image scheduled build for latest tags

https://github.com/mdouchement/standardfile/blob/master/.github/workflows/docker.yml#L4

Maybe using `https://api.github.com/repos/mdouchement/standardfile/releases/latest`

```
error: pathspec 'v0.6.8' did not match any file(s) known to git
Error: Process completed with exit code 1.
```

Forcing `fetch-depth: 0` on [actions/checkout@v2](https://github.com/actions/checkout) should resolve the bug.

--- a/.github/workflows/docker.yml
+++ b/.github/workflows/docker.yml
@@ -12,6 +12,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Fetch all history for all branches and tags
       #
       - name: Prepare
         id: prep

